RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -111,12 +142,23 @@
         constantValueForLostHitsFractionFilter = 0.501,
     )
 )
+eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
+    _detachedQuadStepTrajectoryFilterBase.clone(
+        maxCCCLostHits = 2,
+        maxLostHitsFraction = cms.double(1./10.),
+        constantValueForLostHitsFractionFilter = cms.double(2.0),
+        minGoodStripCharge = dict(refToPSet_ = 'SiStripClusterChargeCutNone')
```

Comment:
Why do you set e.g. `maxCCCLostHits`? I don't see it in https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/Phase2PU140_DetachedQuadStep_cff.py. I would think the following would be
```
eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
  _detachedQuadStepTrajectoryFilterBase.clone(
    maxLostHitsFraction = 1./10.,
    constantValueForLostHitsFractionFilter = 0.301,
  )
)
```